docs/conf/prague/2022/news/welcome.rst

```diff
@@ -0,0 +1,65 @@
+:template: {{year}}/generic.html
+
+.. post:: April 20, 2022
+   :tags: {{shortcode}}-{{year}}
+
+Announcing Write the Docs {{ city }} {{year}}
+===========================================
+
+Welcome to the official launch of `Write the Docs {{ city }} {{year}} <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year }}/>`_.
+We're excited to announce the dates for our next conference: **Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**
+
+We will be running a **virtual event** again this year, same as the Portland conference. 
+We make this very difficult decision with the understanding that it will disappoint some of you.
+We also make this decision with the knowledge that it will relieve some of you from stress or anxiety.
+
+**After talking with our vendors and staff, we determined we couldn't produce the quality of event that our attendees expect if we run a hybrid event.**
+
+If you want to read more about the detailed reasons for keeping our events virtual this year, check out the `Portland conference is going back to virtual <https://www.writethedocs.org/conf/portland/2022/news/conference-going-virtual/>`_ blog post. 
+
+It's been another rough year for a lot of us, especially for our community members in Central and Eastern Europe, but we hope that by the time the conference happens the situation will be calmer and that we'll see many of you join us from wherever you are in the world.
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+**We plan to launch our CFP in early May** for those who are excited to speak.
+All talks will be pre-recorded again, so you'll be able to present from anywhere!
+
+Start thinking about your talk proposal now, and we'll let you know as soon as the CFP opens.
+
+
+Tickets
+-------
+
+**Tickets will go on sale in {{ date.tickets_live }}**.
```

Comment:
This says June but I feel like we can align it with the CFP launch. Thoughts?

Reply:
Sounds good to me